crates/sncast/src/lib.rs

```diff
@@ -245,6 +247,32 @@ pub async fn get_nonce(
 }
 
 pub async fn get_account<'a>(
+    account: &str,
+    accounts_file: &Utf8PathBuf,
+    provider: &'a JsonRpcClient<HttpTransport>,
+    url: &str,
+    keystore: Option<&Utf8PathBuf>,
+    ui: &UI,
+) -> Result<SingleOwnerAccount<&'a JsonRpcClient<HttpTransport>, LocalWallet>> {
```

Comment:
note: I pass url and provider instead of the entire rpc args because this avoids lifetime issues.